test/blackbox-tests/test-cases/quoting/run.t

```diff
@@ -26,9 +26,11 @@ The targets should only be interpreted as a single path when quoted
   lines: foo bar baz
 
   $ dune build @quoted --root filename-space
-  Entering directory 'filename-space'
-  filename contains spaces
+  File "dune", line 4, characters 18-19:
+  Error: Character not allowed inside %{...} forms
```

Comment:
Reading this again, I think the following is even clearer: `This character is not allowed inside %{...} forms`